Cargo.toml

```
@@ -12,6 +12,8 @@ serde = { version = "1", optional = true, default_features = false }
 [dev-dependencies]
 proptest = "0.8.3"
 serde_json = "1"
+serde_derive = "1"
+serde = "1"
```

```suggestion
serde = { versions = "1", features = [ "derive" ] }
```

there's no need to depend on derive separately I thinkg